metricbeat/module/ceph/monitor_health/monitor_health.go

```diff
@@ -61,11 +60,24 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 	}, nil
 }
 
-func (m *MetricSet) Fetch() ([]common.MapStr, error) {
+// Fetch methods implements the data gathering and data conversion to the right
+// format. It publishes the event which is then forwarded to the output. In case
+// of an error set the Error field of mb.Event or simply call report.Error().
+func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 	content, err := m.HTTP.FetchContent()
 	if err != nil {
-		return nil, err
+		return err
+	}
+
+	events, err := eventsMapping(content)
+	if err != nil {
+		m.Logger().Error(err)
```

Comment:
This means it will be logged twice, as error and on the info level. Is that intentional?

Reply:
It was a leftover. Removed now. Thanks!